File: amiqin/shi-config/src/main/java/com/storyteller_f/shi/TitleFilterConfigItem.java
package com.storyteller_f.shi;

import com.storyteller_f.filter_core.config.SimpleRegExpConfigItem;
import org.jetbrains.annotations.NotNull;

public class TitleFilterConfigItem extends SimpleRegExpConfigItem {

    public TitleFilterConfigItem(@NotNull String regexp) {
        super(regexp);
    }
}

File: amiqin/shi-config/src/main/java/com/storyteller_f/shi/TitleFilterConfigItem.kt
package com.storyteller_f.shi

import com.storyteller_f.filter_core.config.SimpleRegExpConfigItem

class TitleFilterConfigItem(regexp: String) : SimpleRegExpConfigItem(regexp) {
    override fun dup(): Any {
        return TitleFilterConfigItem(regexp)
    }
}

class UrlFilterConfigItem(url: String) : SimpleRegExpConfigItem(url) {
    override fun dup(): Any {
        return UrlFilterConfigItem(regexp)
    }

}
